packages/connection/src/class-manager.php

```diff
@@ -2127,9 +2127,9 @@ public function authorize( $data = array() ) {
 			return new \WP_Error( 'no_token', 'Error generating token.', 400 );
 		}
 
-		$is_master_user = ! $this->is_active();
+		$is_connection_owner = false === $this->get_connection_owner();
```

Comment:
The `Manager` class has 2 methods that might be more appropriate here (not sure)

`is_connection_owner` - which has the same name as the variable, and checks if the current user is the connection owner

`has_connection_owner` - that only checks if the connection is owned by someone and returns a boolen... it's basically the same thing you're doing

Reply:
Good point, fixed now.
Please re-review.